packages/venia-ui/lib/components/AccountInformationPage/editForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { Fragment } from 'react';
+import { func, shape, string, bool } from 'prop-types';
+
+import { mergeClasses } from '../../classify';
+import Field from '../Field';
+import TextInput from '../TextInput';
+import LinkButton from '../LinkButton';
+import Password from '../Password';
+import combine from '../../util/combineValidators';
+import {
+    hasLengthAtLeast,
+    isRequired,
+    validatePassword,
+    validateDifferentCurrentPassword
+} from '../../util/formValidators';
+import defaultClasses from './editForm.css';
+
+const EditForm = props => {
+    const {
+        classes: propClasses,
+        handleActivePassword,
+        activeChangePassword
```

Comment:
The `handleActivePassword` and `activeChangePassword` props seem like they could belong to the edit form alone rather than the account page. Was there some logic or state that the parent `AccountInformationPage` needs that it couldn't get from the edit form?